logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageRepository.kt

```diff
@@ -151,6 +149,17 @@ interface MessageRepository {
         conversationId: ConversationId
     ): Either<CoreFailure, Conversation.ReceiptMode?>
 
+    /**
+     * updates the message status to [MessageEntity.Status.SENT] and the server date to [serverDate]
+     * also mark other pending messages and add millis to their date
+     */
+    suspend fun updateMessagesAfterOneIsSent(
+        conversationId: ConversationId,
+        messageUuid: String,
+        serverDate: Instant,
+        millis: Long
```

Comment:
what's millis? the time difference between server and local? or the latest one?

Reply:
I know the it was already named like this for it is very unclear what it does. Holds milliseconds of something? 

sendDuration
deliveryDuration

It could also be an actual `Duration`.

Reply:
> yes
I assume this answer will be regarding the time difference seeing Jacob's comment above. Then I guess `durationInMillis` could be a more appropriate naming 😅